lms/static/js/components/card/views/card.js

```diff
@@ -35,25 +36,25 @@
                  * depends on this.configuration being set to pick the appropriate class. Therefore, configuration
                  * is set in the constructor, but the rest of the initialization happens in initialize.
                  */
-                constructor: function (options) {
+                constructor: function(options) {
                     if (!this.configuration)  {
                         this.configuration = (options && options.configuration) ? options.configuration : 'square_card';
                     }
                     Backbone.View.prototype.constructor.apply(this, arguments);
                 },
 
-                initialize: function () {
+                initialize: function() {
                     this.render();
                 },
 
-                template: _.template(cardTemplate),
+                template: HtmlUtils.template(cardTemplate),
```

Comment:
Are we just switching to HtmlUtils.template for consistency, or is there an actual need here?